src/components/RewardPool.ts

```diff
@@ -158,6 +174,27 @@ export default class RewardPool {
     return base;
   };
 
+  private getDataPoolRewardsMulticall = (
+    account: string | undefined,
+    signer: ethers.Signer,
+    pools: Record<string, number>
+  ): Promisified<{ [poolName: string]: BigNumber }> => {
+    if (!account) throw new Error("No account provided");
+
```

Comment:
There is no purpose in allowing `account` to be undefined if all the function does is throw if it is undefined.
```suggestion
  private getDataPoolRewardsMulticall = (
    account: string,
    signer: ethers.Signer,
    pools: Record<string, number>
  ): Promisified<{ [poolName: string]: BigNumber }> => {
```

Reply:
refactored